tests/autologging/test_autologging_safety_unit.py

```diff
@@ -1141,16 +1141,16 @@ def test_validate_args_succeeds_when_arg_sets_are_equivalent_or_identical():
         "biz": {"baz": 5},
     }
 
-    _validate_args(args, kwargs, args, kwargs)
-    _validate_args(args, None, args, None)
-    _validate_args(None, kwargs, None, kwargs)
+    _validate_args("", "", args, kwargs, args, kwargs)
```

Comment:
Nit: Can we pass the strings `"autologging_integration_name"` and `"function_name"` as arguments, rather than empty strings, so that maintainers know how to interpret the purpose / meaning of these args?